src/components/panel/panel.js

```diff
@@ -865,15 +866,16 @@ MdPanelRef.prototype.close = function() {
         return self.detach();
       }, this._$q.reject)
       .then(function() {
-        // TODO(ErinCoughlan) - Add destroy. This will make the code here
-        // different than just calling this.detach().
+        var origin = self._config['origin'];
```

Comment:
Should this be part of hide instead of close? Otherwise, the user would need to manually focus whenever the panel is hidden, which is what this is avoiding.